packages/block/src/block.ts

```diff
@@ -1,7 +1,7 @@
 import Common from 'ethereumjs-common'
 import * as ethUtil from 'ethereumjs-util'
 import { BN, rlp } from 'ethereumjs-util'
-import { Transaction } from 'ethereumjs-tx'
+import { Transaction, TransactionOptions } from 'ethereumjs-tx'
```

Comment:
We need `TransactionOptions` interface [here](https://github.com/ethereumjs/ethereumjs-vm/pull/682/files#diff-61ac76e0777f844b544ffa824f6ccbdfR82).